templates/data/opensearch/managed.html

```diff
@@ -0,0 +1,447 @@
+{% extends 'base_index.html' %}
+
+{% block meta_copydoc %}
+  https://docs.google.com/document/d/19gcIoKEibXTvZCsmtYCSQBMnBUmNUJI6RaLoMHL2_TA/edit
+{% endblock %}
+
+{% block title %}Managed OpenSearch services by Canonical{% endblock %}
+
+{% block meta_description %}
+  Learn about Canonical’s managed services for OpenSearch, the industry leading open source search engine, analytics tool and vector database.
+{% endblock %}
+
+{% block body_class %}is-paper{% endblock %}
+
+{% block content %}
+  <section class="p-section--hero">
+    <div class="row">
+      <div class="col-3 col-medium-2">
+        <div class="u-hide--small">
+          {{ image(url="https://assets.ubuntu.com/v1/ff455ff8-opensearch-logo--signpost.png",
+                    alt="OpenSearch",
+                    width="210",
+                    height="70",
+                    hi_def=True,
+                    loading="lazy") | safe
+          }}
+        </div>
+      </div>
+      <div class="col-9 col-medium-4">
+        <div class="p-section--shallow">
+          <h1>Managed OpenSearch by Canonical</h1>
+          <p>
+            Free up time to focus on building applications and business development rather than maintaining your database. Get automation, enterprise-grade security, scalability, availability, monitoring, and support from trusted experts with our Managed OpenSearch service.
+          </p>
+        </div>
+        <hr class="is-muted" />
+        <a class="p-button--positive"
+           href="/contact-us"
+           aria-controls="data-opensearch-modal">Schedule a call</a>
```

Comment:
This button doesn't work

Reply:
you're right, sorry, totally missed that. Will push a fix. Thanks for the review!